app/controllers/provider_foreman_controller.rb

```diff
@@ -982,6 +1054,59 @@ def add_unassigned_configuration_profile_record_to_view(unassigned_profile_row,
     @grid_hash = view_to_hash(@view)
   end
 
+  def unassigned_inventory_group?(node)
+    _type, _pid, nodeinfo = node.split("_").last.split("-")
+    nodeinfo == "unassigned"
+  end
+
+  def empty_inventory_group_record?(inventory_group_record)
+    inventory_group_record.try(:id).nil?
+  end
+
+  def valid_inventory_group_record?(inventory_group_record)
+    inventory_group_record.try(:id)
+  end
+
+  def inventory_group_right_cell_text(model)
+    record_model = ui_lookup(:model => model ? model : TreeBuilder.get_model_for_prefix(@nodetype))
+    return if @sb[:active_tab] != 'configured_systems'
+    if valid_inventory_group_record?(@inventory_group_record)
+      @right_cell_text = _("%{model} under Inventory Group \"%{name}\"") %
+                           {:model        => ui_lookup(:tables => "configured_system"),
+                            :record_model => record_model,
+                            :name         => @inventory_group_record.name}
+    else
+      @right_cell_text = _("%{model} under Unassigned Inventory Group") %
```

Comment:
I don't think you can have a configured system without an inventory_root_group.  Have you found a case where that is true?